### PR TITLE
AST2-192: measures get filtered based on selected geometry type

### PR DESCRIPTION
**Pull request by Hornebom**
Progress: All available measures get filtered based on selected geometry. If no selection, all measures are shown.

Issues: 
- Vue devtools crashed after multiple clicks on different areas
- Vue devtools don't update state of project.areas, but the filter logic seems to work (???)
- dato isn't updated regarding draw types >> so I tested the length of filtered measures

### CODE REVIEW

File: src/client/components/measure-list/measure-list.vue [hunk 1]
```diff
@@ -70,15 +71,36 @@ export default {
...
+        ? this.filterMeasureByType(types[type])
+        : this.sortedMeasures;
+
+
+      return availableMeasures.filter(measure => {
         return measure.title.match(new RegExp(this.searchValue, 'i'))
       })
+      // return this.sortedMeasures.filter(measure => {
+      //   return measure.title.match(new RegExp(this.searchValue, 'i'))
+      // })
```
Comment (petergoes) [inline src/client/components/measure-list/measure-list.vue]: Please remove the commented out lines

File: src/client/components/measure-list/measure-list.vue [hunk 2]
```diff
@@ -70,15 +71,36 @@ export default {
...
     filteredMeasures() {
-      return this.sortedMeasures.filter(measure => {
+      const types = {
+        'Polygon': 'canDrawPolygon',
+        'LineString': 'canDrawPolyline',
+        'Point': 'canDrawPoint',
+      }
+      const type = this.selectedType();
+      const availableMeasures = type
+        ? this.filterMeasureByType(types[type])
```
Comment (petergoes) [inline src/client/components/measure-list/measure-list.vue]: `type` refers to the currently selected type. On L79 you reference it from `selectedType`. Because we know what type we are dealing with (the _selected_ type) we can make the statement `types[type]` more explicit by doing: `types[selectedType]`. This way we indicate that from the `types` array we want the current selected type, not some random type.

tldr; `types[type]` => `types[selectedType]`
